### PR TITLE
[DT-45] make the app view layout according to the figma requirements.

### PR DESCRIPTION
**Pull request by jdandrade**
**What does this PR do?**

   Improvements to the App View UI to match the figma design. Some changes were to add flexibility to the common UI code in order to allow different ui implementation for the gameshub product.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppViewView.kt

**How should this be manually tested?**

  See if UI matches figma files and info in ticket + UI notes from confluence (all in ticket)

**What are the relevant tickets?**

  Tickets related to this pull-request: [DT-45](https://aptoide.atlassian.net/browse/DT-45)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

### CODE REVIEW

File: feature_appview/src/main/java/cm/aptoide/pt/feature_appview/data/AptoideAppViewRepository.kt [hunk 1]
```diff
@@ -84,16 +85,26 @@ class AptoideAppViewRepository @Inject constructor(
     return flow {
       val relatedContentResponse = remoteAppViewRepository.getRelatedContent(packageName)
       if (relatedContentResponse.isSuccessful) {
-        relatedContentResponse.body()?.datalist?.list?.let {
-          emit(RelatedContentResult.Success(it.map { relatedCardJson -> relatedCardJson.toDomainModel() }))
+        relatedContentResponse.body()?.datalist?.list?.map { relatedCard ->
+          reactionsRepository.getTotalReactions(relatedCard.id)
```
Comment (G0retZ) [inline feature_appview/src/main/java/cm/aptoide/pt/feature_appview/data/AptoideAppViewRepository.kt:89]: I already refactored reactions a bit, so you can use that

File: app/src/main/java/cm/aptoide/pt/appview/AptoideTabsListProvider.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+class AptoideTabsListProvider : TabsListProvider {
+  override fun getTabsList(): List<Pair<AppViewTab, Int>> =
+    listOf(
+      Pair(AppViewTab.DETAILS, 0),
+      Pair(AppViewTab.REVIEWS, 1),
+      Pair(AppViewTab.RELATED, 2),
+      Pair(AppViewTab.VERSIONS, 3),
+      Pair(AppViewTab.INFO, 4)
+    )
+}
```
Comment (G0retZ) [inline app/src/main/java/cm/aptoide/pt/appview/AptoideTabsListProvider.kt:17]: New line at EOF is missing

**Review by G0retZ — CHANGES_REQUESTED** [review]
Better keep old formatting for sake of readability

File: app/src/main/java/cm/aptoide/pt/appview/AptoideTabsListProvider.kt (hunk 2 above)
Comment (jdandrade) [reply app/src/main/java/cm/aptoide/pt/appview/AptoideTabsListProvider.kt]: Done

File: feature_appview/src/main/java/cm/aptoide/pt/feature_appview/data/AptoideAppViewRepository.kt (hunk 1 above)
Comment (jdandrade) [reply feature_appview/src/main/java/cm/aptoide/pt/feature_appview/data/AptoideAppViewRepository.kt:89]: I can only use that after your PR gets merged or already in code?

**Pull request merged**